.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2
```

Comment:
I'm not familiar with version 2 of CircleCI so I can't make particularly useful comments here. I assume it's working though!

Reply:
The main difference in CircleCI version 2 is the `workflow` section, which defines how different jobs should be launched (including job dependencies). We have only one job `tests` right now. But later will probably add another job like `deployment` to enable continuous deployment on AWS.